### ISSUE TITLE
Incorrect Instructions

### ISSUE THREAD

**Issue by kevinchandler** [FreeCodeCamp/FreeCodeCamp#3713]
The instructions state to remove all values from the array greater than 5. The provided answer does just that, however the answer being looked for is an array with all values greater than ***or equal to*** 5 removed.  

---

Challenge [Waypoint: Filter Arrays with filter](http://freecodecamp.com/challenges/waypoint-filter-arrays-with-filter#?solution=var%20array%20%3D%20%5B1%2C2%2C3%2C4%2C5%2C6%2C7%2C8%2C9%2C10%5D%3B%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%0Aarray%20%3D%20array.filter(function(val)%20%7B%0A%E2%80%89%E2%80%89return%20val%20%3C%205%3B%0A%7D)%3B%0A%0A%20%20%2F%2F%20Only%20change%20code%20above%20this%20line.%0A(function()%20%7Breturn%20array%3B%7D)()%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.101 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
var array = [1,2,3,4,5,6,7,8,9,10];
  // Only change code below this line.

array = array.filter(function(val) {
  return val < 5;
});

  // Only change code above this line.
(function() {return array;})();

```

**Comment by jlslew** [comment]
Thanks for posting.  There is a similar issue; #3671 and a fix has been accepted; #3678.

**Comment by bugron** [comment]
Closing this issue as it is fixed in https://github.com/FreeCodeCamp/FreeCodeCamp/pull/3678 and, hopefully, the fix should be in production soon.
